training/src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -72,6 +78,19 @@ public User create(@RequestBody User user) throws NullAttributesException {
         if(user.anyRequiredAttributeNull()){
             throw new NullAttributesException();
         }
+
+        user.setPassword(passwordEncoder.encode(user.getPassword()));
+
+        return userRepository.save(user);
+    }
+
+    @PutMapping("/editPass/{userId}")
+    public User updatePassword(@PathVariable Long userId, @RequestBody User u)
+        throws UserNotFoundException {
+        String password = u.getPassword();
+        User user = userRepository.findById(userId).orElseThrow(UserNotFoundException::new);
+        user.setPassword(passwordEncoder.encode(password));
```

Comment:
This encryption wouldn't be necessary if you add the encryption inside the setter.